<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoRunDlgClass</class>
 <widget class="QDialog" name="AutoRunDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoRunDlg</string>
  </property>
  <widget class="QPushButton" name="runBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="pathTab">
    <attribute name="title">
     <string>path</string>
    </attribute>
    <widget class="QToolButton" name="projectOpenBtn">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>100</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>las path</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>result folder</string>
     </property>
    </widget>
    <widget class="QToolButton" name="lasOpenBtn">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="lasPath">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="resultFolder">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label1_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>point desity of resample</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="pointDensity">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>170</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="cellSize">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label1_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>cell size of grid index</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="filteringTab">
    <attribute name="title">
     <string>seed generation</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox2_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>611</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>threshold for scene complex detection and seeds generation</string>
     </property>
     <widget class="QTextEdit" name="seedSlopeTh">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="seedHeightTh">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label2_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>slope threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="label2_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>height threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="label2_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>small seed cell size</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="seedCellSize">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>130</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="useOBASeedGeneratorCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>using advanced
seeds generator</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextEdit" name="maxObjSize">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label2_1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>max object size</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>filtering</string>
    </attribute>
    <widget class="QLabel" name="label3_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>local height threshold</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="hIteration">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label3_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>angle iteration</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="angleIteration">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="filteringSlopeTh">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>90</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label3_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>local angle threshold</string>
     </property>
    </widget>
    <widget class="QLabel" name="label3_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>height iteration</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="filteringHeightTh">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label3_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dtm resolution factor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label3_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>160</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>object height threshold</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="dtmFactor">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="objectHT">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="extractionTab">
    <attribute name="title">
     <string>classification</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>building extraction</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>information</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
